src/elements/file-selector/file-selector.visual.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { html, nothing } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../core/testing/private.js';
+
+import '../form-error.js';
+import './file-selector.js';
+import type { SbbFileSelectorElement } from '@sbb-esta/lyne-elements/file-selector/file-selector';
+
+describe(`sbb-file-selector`, () => {
+  function addFilesToComponentInput(elem: SbbFileSelectorElement): void {
+    const dataTransfer: DataTransfer = new DataTransfer();
+    for (let i: number = 0; i < 5; i++) {
+      dataTransfer.items.add(
+        new File([`Hello world - ${i}`], `hello${i}.txt`, {
+          type: 'text/plain',
+          lastModified: new Date(i).getMilliseconds(),
+        }),
+      );
+    }
+    const input: HTMLInputElement = elem.shadowRoot!.querySelector<HTMLInputElement>('input')!;
+    input.files = dataTransfer.files;
+    input.dispatchEvent(new Event('change'));
+  }
+
+  let root: HTMLElement;
+
+  const states = {
+    variant: ['default', 'dropzone'],
+    state: [
+      { disabled: false, error: false },
+      { disabled: true, error: false },
+      { disabled: false, error: true },
+    ],
+  };
+
+  const sizes = {
+    variant: ['default', 'dropzone'],
+    size: ['s', 'm'],
+  };
+
+  describeViewports({ viewports: ['small', 'medium'] }, () => {
```

Comment:
```suggestion
  describeViewports({ viewports: ['zero', 'medium'] }, () => {
```
Usually, we use zero/medium

Reply:
Yes, but in zero the file's icons are cropped, so i went with small
https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-file-selector--default&globals=backgrounds.grid:!false;viewport:zero